guides/plugins/plugins/checkout/cart/use-nested-line-items.md

```diff
@@ -0,0 +1,41 @@
+# Use nested line items
+
+# Overview
+
+This guide will show you how to use the nested line items in the storefront.
+
+# Prerequisites
+
+As most guides, this guide is also built upon the [Plugin base guide](../../plugin-base-guide.md), but you don't necessarily need that. It will only extend views and will show how the Custom Product plugin handle this.
+
+# Make nested line item removable
+
+If the nested line item should removable in the cart, the removable property has to be set (via view or in the own controller action) and a form with an own path action has to be added:
+
+```
+{% block page_checkout_item_remove_icon %}
+    {% do nestedLineItem.setRemovable(true) %}
+    <form action="{{ path('/mycontroller/nested/remove', {'id': nestedLineItem.id}) }}" method="post">
+        {{ parent() }}
+    </form>
+{% endblock %}
+```
+
+# Make nested line item changeable
+
+Most of the time the root line item defines the nested line items, because of this there is a change button for the root line item in the cart.
+In the block of the change button, the variable `isChangeable` has to be set, and the button has to be surrounded with a link to the action like this:
+
+```
+{% block component_offcanvas_item_children_header_content_change_button %}
+    {% set isChangeable = true %}
+    <a href="{{ seoUrl('frontend.detail.page', {
+    'productId': lineItem.children.first.referencedId,
+    'swagCustomizedProductsConfigurationEdit': lineItem.extensions.customizedProductConfiguration.id
```

Comment:
Würde ich entweder auslagern, oder - noch besser - du legst vorher ne variable mit dem content an. Das machts sonst sehr unleserlich.

Reply:
Zumindest ein Kommentar, warum du das nicht gemacht hast, wär lieb :D

Reply:
Habe ich doch gemacht ;) Schau mal in die aktuelle Änderung (siehe Label Outdated ;)